sources/react-app/src/shared/BaseLayout.js

```diff
@@ -19,11 +19,11 @@ import Footer from '../components/Footer';
 import React from 'react';
 import { useGlobalContext } from './context';
 
-export default function BaseLayout({ children }) {
+export default function BaseLayout({ pageTitle, children }) {
   const [{ theme }] = useGlobalContext();
   return (
     <div className={`wrap theme-${theme}`}>
-      <Header />
+      <Header pageTitle={pageTitle}/>
```

Comment:
This is not the _pageTitle_. It is the site's name (siteName) — or site title (siteTitle). Change all over. What's the name in the level descriptor?